integration/commands_administration_test.go

```diff
@@ -563,17 +565,19 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
 				Name:    "TypeMismatch",
 				Message: `BSON field 'getParameter.allParameters' is the wrong type 'string', expected types '[bool, long, int, decimal, double']`,
 			},
+			altMessage: `BSON field 'allParameters' is the wrong type 'string', expected types '[bool, long, int, decimal, double]'`,
 		},
 	} {
 		name, tc := name, tc
 		t.Run(name, func(t *testing.T) {
-			t.Parallel()
+			//	t.Parallel()
 
 			var actual bson.D
 			err := collection.Database().RunCommand(ctx, tc.command).Decode(&actual)
 
 			if tc.err != nil {
-				AssertEqualError(t, *tc.err, err)
+				AssertEqualAltError(t, *tc.err, tc.altMessage, err)
+				//				AssertEqualError(t, *tc.err, err)
```

Comment:
There are should be no dead code. We have GitHub for this.

Reply:
sorry, four parallel tasks in development is bad
Thanks